### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by cufezhusy** [ososib/Deep-Hedging-of-CVA#3] [General]
Hello, how is everything going?

**Comment by ososib** [comment]
Hi,

We have fixed the code for the market simulations from last week if you want to have a look.  We have also spent some time recently working at the formal "Thesis Purpose Statement", which now has been approved by our examiner (see attached). Maybe it is interesting for you to look at our objective and methodology, and give feedback if necessary. 
[Måldokument-2.pdf](https://github.com/ososib/Deep-Hedging-with-cross-gamma/files/14190496/Maldokument-2.pdf.

We have been also thinking of hedging scenarios in our model. What assumptions seem reasonable to have in our current model?

Some thoughts on our simulated market:
1. Which instruments are to be used as a hedge? Are processes A and B good enough and are they supposed to be "traded"?
2. Our intuition led us to that CDS is the only way to hedge CVAs (credit component)? Does including CDS help us in this case? Or does only complicate things? We are thinking of appropriate ways to include the CDS if appropriate. 

We will discuss these questions with our finance professor.

I suggest we have a short meeting soon to discuss above. We could do it whenever tomorrow or today. And Friday before 11. Otherwise some time next week.

**Reply by cufezhusy** [reply]
Hi,

It is very good that you ask those questions. 

The general idea when you define variable P = A*B,  P is an approximation of CVA (itself), A and B is two instruments people can used to hedge. One of the A and B is CDS as you pointed out.

Let me know if above is clear, if not please let me know.

BR
Shengyao

**Comment by ososib** [comment]
Hi,

Thank you! We still have more questions about our considered model. Is it possible to have a short meeting tomorrow, Thursday or Friday?

Best,
Osman & Oscar

**Comment by cufezhusy** [comment]
Sure, I booked you sometime this afternoon. Hope it works?

**Comment by ososib** [comment]
Hi Shengyao,

Hope all is well:)

Do you have any suggestions for a meeting time/date in Copenhagen soon? We are flexible, suggest some time you are available and we will try to decide based on that!:D

Happy Weekend! :)

**Comment by cufezhusy** [comment]
Hi! I will be off from 11 March. Before that, I can do it in next Friday (23 Feb) of the Friday after next week (1 Mar). Both morning and afternoon should work.

Have a nice weekend :)

BR
Shengyao

**Comment by ososib** [comment]
My girlfriend is turning 26 on the 23th Feb so I should be at Lund and celebrate her birthday:D

Let's do 1 March afternoon, both me and Oscar are available. Would be nice to meet you:)

**Comment by cufezhusy** [comment]
That sounds perfect! See u on 1 March then :)

**Comment by ososib** [comment]
Hi Shengyao,

Thanks for your friendly meeting last Friday at Markets! It was nice to meet you!:)

Small question regarding metrics/rewards to sought after. Are the metrics VAR and ES relevant for the PnL of XVA derivatives? I am wondering if you do this already at your desk?

**Comment by cufezhusy** [comment]
Hello!
Thanks for the visiting!
Yes VaR and ES will also be a good way to measure the variance of hedging performance, however it is not widely used in a front office environment, because typically the measure is 'un-realized' and model driven. What we have also been seen is second line impose some limit to trading desk on VaR and ES, so it can be used as a constraint for the optimization.
BR
Shengyao

**Comment by ososib** [comment]
Hi Shengyao,

When should we meet again? You are probably at home now? Enjoy your time off and let us know when you are available again:)
